### ISSUE TITLE
Relocated wow cant find new folder

### ISSUE THREAD

**Issue by s19locke** [WeakAuras/WeakAuras-Companion#506]
**Operating System:**
- [ X] Windows
- [ ] Mac
- [ ] Linux

**Companion Version:**
```
Check the bottom left of the "About" panel in the app.
```

**Companion App Console Log:**
```
Press Ctrl+Shift+I to open the console and copy-paste everything here
```
_first off I couldn't get ^ to work,_  _secondly the bug is I recently moved wow from a ssd to an NVME drive. upon doing so the companion app lost track of the wow folder and it wont let me relocate it_

![image](https://user-images.githubusercontent.com/58767049/70602983-a0e29d00-1bc3-11ea-918c-02ce55897553.png)

![image](https://user-images.githubusercontent.com/58767049/70603138-e30bde80-1bc3-11ea-8940-e46ed33b00f6.png) 

_after clicking the select folder button nothing changed still says I:\World of Warcraft_

_I have tried uninstalling the program and re-installing it. but apparently there is some saved data somewhere that the un-install doesn't get rid off because upon re-install it still tries to find wow in the I:\ drive which doesn't exist and still will not let me change it_

Addin … is Crtl + shift + I supposed to open anything ? its not... but for kicks and giggles I tried it again followed by a crtl+C and got this
`<body><noscript><strong>We're sorry but weakauras-companion doesn't work properly without JavaScript enabled. Please enable it to continue.</strong></noscript><div id="app"><div id="wrapper"><div class="main-container"><div data-v-318568de="" id="ui-titlebar"><div data-v-318568de="" id="ui-titletext"></div><div data-v-318568de="" id="ui-titlecontrols"><button data-v-318568de="" class="ui-btn minimize"><svg data-v-318568de="" x="0px" y="0px" viewBox="0 0 10.2 1"><rect data-v-318568de="" x="0" y="50%" width="10.2" height="1"></rect></svg></button><button data-v-318568de="" class="ui-btn close"><svg data-v-318568de="" viewBox="0 0 10 10"><polygon data-v-318568de="" points="10.2,0.7 9.5,0 5.1,4.4 0.7,0 0,0.7 4.4,5.1 0,9.5 0.7,10.2 5.1,5.8 9.5,10.2 10.2,9.5 5.8,5.1"></polygon></svg></button></div></div><header><div class="app-logo"><img src="app://./img/weakauras.65effd90.png" class="logo-img"><span>Companion</span></div><div class="menu-btns"><button class="btn btn-menu">Main</button><button class="btn btn-menu active">Settings</button><button class="btn btn-menu">Help</button><button class="btn btn-menu">About</button></div></header><main><!----><!----><div data-v-7fa17ae5="" id="config"><div data-v-7fa17ae5="" class="config-row"><div data-v-7fa17ae5="" class="config-row-item"><div data-v-7fa17ae5="" class="title"> Game Settings </div><div data-v-7fa17ae5="" class="block"><label data-v-d12945f2="" data-v-7fa17ae5="" class="file-select"><span data-v-d12945f2="" class="select-button"><p data-v-d12945f2="" class="configlabel"> World of Warcraft Folder </p><div data-v-d12945f2="" class="fakeinput pointer"><span data-v-d12945f2="" class="wow-path">I:\World of Warcraft </span></div><i data-v-d12945f2="" class="material-icons folder">folder_open</i></span></label><i data-v-7fa17ae5="" class="material-icons red folder">error_outline</i></div><div data-v-7fa17ae5="" class="title"> Companion Settings </div><div data-v-7fa17ae5="" class="block"><div data-v-137fdb63="" data-v-7fa17ae5="" class="dropdown"><label data-v-137fdb63="" class="dropdown__label"> Language </label><div data-v-137fdb63="" class="dropdown__toggle"><span data-v-137fdb63="">English (en)</span><i data-v-137fdb63="" class="material-icons"> keyboard_arrow_down </i></div><div data-v-137fdb63="" class="dropdown__options" style="height: 0px;"><div data-v-137fdb63="" title="中文 (简体) (zh-cn)" class="dropdown__option"><span data-v-137fdb63="">中文 (简体) (zh-cn)</span></div><div data-v-137fdb63="" title="Deutsch (de)" class="dropdown__option"><span data-v-137fdb63="">Deutsch (de)</span></div><div data-v-137fdb63="" title="English (en)" class="dropdown__option"><span data-v-137fdb63="">English (en)</span></div><div data-v-137fdb63="" title="Español (es)" class="dropdown__option"><span data-v-137fdb63="">Español (es)</span></div><div data-v-137fdb63="" title="Français (fr)" class="dropdown__option"><span data-v-137fdb63="">Français (fr)</span></div><div data-v-137fdb63="" title="Русский (ru)" class="dropdown__option"><span data-v-137fdb63="">Русский (ru)</span></div><div data-v-137fdb63="" title="Türkçe (tr)" class="dropdown__option"><span data-v-137fdb63="">Türkçe (tr)</span></div></div></div><div data-v-07d6eb08="" data-v-7fa17ae5="" class="checkbox"><input data-v-07d6eb08="" type="checkbox"><div data-v-07d6eb08="" class="checkbox__box checkbox__box--checked"></div><label data-v-07d6eb08=""> Show auras without updates </label></div><div data-v-07d6eb08="" data-v-7fa17ae5="" class="checkbox"><input data-v-07d6eb08="" type="checkbox"><div data-v-07d6eb08="" class="checkbox__box checkbox__box--checked"></div><label data-v-07d6eb08=""> Show notifications for new updates </label></div><p data-v-7fa17ae5="" class="label subtitle"> Startup </p><div data-v-7fa17ae5="" class="option"><div data-v-07d6eb08="" data-v-7fa17ae5="" class="checkbox"><input data-v-07d6eb08="" type="checkbox"><div data-v-07d6eb08="" class="checkbox__box checkbox__box--checked"></div><label data-v-07d6eb08=""> Launch client with your computer </label></div></div><div data-v-7fa17ae5="" class="option"><div data-v-07d6eb08="" data-v-7fa17ae5="" class="checkbox"><input data-v-07d6eb08="" type="checkbox"><div data-v-07d6eb08="" class="checkbox__box"></div><label data-v-07d6eb08=""> Start client minimized </label></div></div><p data-v-7fa17ae5="" class="label subtitle"> Auto-Update </p><div data-v-7fa17ae5="" class="option"><div data-v-07d6eb08="" data-v-7fa17ae5="" class="checkbox"><input data-v-07d6eb08="" type="checkbox"><div data-v-07d6eb08="" class="checkbox__box checkbox__box--checked"></div><label data-v-07d6eb08=""> Use Companion Beta channel </label></div></div><div data-v-7fa17ae5="" class="option"><div data-v-07d6eb08="" data-v-7fa17ae5="" class="checkbox"><input data-v-07d6eb08="" type="checkbox"><div data-v-07d6eb08="" class="checkbox__box"></div><label data-v-07d6eb08=""> Install Companion updates automatically </label></div></div></div></div><div data-v-7fa17ae5="" class="config-row-item"><div data-v-7fa17ae5="" class="title"> Wago Settings </div><div data-v-7fa17ae5="" class="block"><p data-v-7fa17ae5="" class="label"> Set Wago Account (optional) </p><input data-v-7fa17ae5="" type="text" size="11"><button data-v-7fa17ae5="" class="btn btn-ok"> OK </button><!----><p data-v-7fa17ae5="" class="label"> Set Wago API Key (optional) </p><input data-v-7fa17ae5="" type="password" size="11"><button data-v-7fa17ae5="" class="btn btn-ok">OK</button><!----><p data-v-7fa17ae5=""><a data-v-7fa17ae5="" href="https://wago.io/account" target="_blank" class="explorer">Get yours</a></p><div data-v-07d6eb08="" data-v-7fa17ae5="" class="checkbox"><input data-v-07d6eb08="" type="checkbox"><div data-v-07d6eb08="" class="checkbox__box"></div><label data-v-07d6eb08=""> Ignore auras from your account </label></div></div><div data-v-7fa17ae5="" class="title"> WeakAuras Backup </div><div data-v-7fa17ae5="" class="block"><p data-v-7fa17ae5="" class="label"><div data-v-07d6eb08="" data-v-7fa17ae5="" class="checkbox"><input data-v-07d6eb08="" type="checkbox"><div data-v-07d6eb08="" class="checkbox__box"></div><label data-v-07d6eb08=""> Activate </label></div></p><!----></div></div></div><div data-v-7fa17ae5="" class="block resetbutton"><button data-v-7fa17ae5="" class="btn btn-reset"> Reset Settings and Data </button></div></div><!----><!----></main><footer><a href="https://www.curseforge.com/wow/addons/weakauras-2" target="_blank" class="getweakauras"><img src="app://./img/curse.33146bce.svg" title="CurseForge" class="logo"> Get WeakAuras! </a><a href="https://wago.io/weakauras" target="_blank" class="browsewago"><img src="app://./img/wago.d03ce305.svg" title="Wago" class="logo"> Browse Wago for more auras! </a><a class="reportbug"> Found a bug? <img src="app://./img/bug_report.ccec199f.svg" title="Bug" class="logo invert"></a><div class="app-update"><!----><!----><!----><!----></div></footer></div><!----><!----></div></div><script type="module" src="app://./js/chunk-vendors.3a6a7bdc.js"></script><script type="module" src="app://./js/app.2eefd980.js"></script><script>!function(){var e=document,t=e.createElement("script");if(!("noModule"in t)&&"onbeforeload"in t){var n=!1;e.addEventListener("beforeload",function(e){if(e.target===t)n=!0;else if(!e.target.hasAttribute("nomodule")||!n)return;e.preventDefault()},!0),t.type="module",t.src=".",e.head.appendChild(t),t.remove()}}();</script><div id="rhtPO7DN9" role="status" aria-live="polite" aria-atomic="false"></div></body>`

**Comment by s19locke** [comment]
Further playing around uninstalled again found the computer thought I had 1.1.5 installed but couldn't find it. Found the setup files for it ran those could successfully located wow folder and update addons again. The app kept pushing to update to 2.0 I did and everything broke again uninstalled the apps, re installed 1.1.5 again leaving it there for now I guess

**Comment by mrbuds** [comment]
I can't reproduce the issue and i don't have a fix yet, but as a workaround you can try replacing `%appdata%/weakauras-companion/config.json` with this file https://cdn.discordapp.com/attachments/532314347477925891/658477597205856278/config_-_Copy.zip rename it, and edit the path of you wow folder

**Comment by mrbuds** [comment]
A fix for this issue was implemented in the last beta version, you can get it by checking this option in the "Settings" tab
![img](https://i.imgur.com/aSCrvZb.png)

**Comment by s19locke** [comment]
![image](https://user-images.githubusercontent.com/58767049/71402261-684ab680-25fa-11ea-9455-b476380c6558.png)

Anyway I'm on 2.0 and have the correct directory now
 The companion kept forcing me to update to 2.0 regardless if I told it to or not. Even when I had the correct path selected in 1.1.5 once it went to 2.0 it broke again. So I uninstalled it again, I found that you have previous versions on here so I installed 1.9 found the correct directory and thankfully it still works now even though its @ 2.0

I apologize if I'm giving your irrelevant data, just trying to help if I can, this App is too amazing to just go its fixed and ignore it.  Just to double check, did you see that I edited the first message and was able to get something from CTRL + Shift + I? I just couldn't see it ... I was able to copy and paste it I think.
